sql/core/src/main/scala/org/apache/spark/sql/execution/python/UserDefinedPythonFunction.scala

```diff
@@ -50,6 +50,19 @@ case class UserDefinedPythonFunction(
     udfDeterministic: Boolean) {
 
   def builder(e: Seq[Expression]): Expression = {
+    if (pythonEvalType == PythonEvalType.SQL_BATCHED_UDF
+        || pythonEvalType ==PythonEvalType.SQL_ARROW_BATCHED_UDF
+        || pythonEvalType == PythonEvalType.SQL_SCALAR_PANDAS_UDF) {
+      /*
+       * Check if the named arguments:
+       * - don't have duplicated names
+       * - don't contain positional arguments
```

Comment:
```suggestion
       * - don't contain positional arguments after named arguments
       * - all map to valid argument names from the function declaration
```

Reply:
The third item is not right for Python UDF/UDTF. Currently it relies on Python.